varexport/src/test/java/com/indeed/util/varexport/VarExporterTest.java

```diff
@@ -121,10 +121,10 @@
         @Export(name="map", expand=true, cacheTimeoutMs=1L)
         public Map<String, Map<String, String>> map = Maps.newHashMap();
```

Comment:
I feel like this should also be a linked hash map for the sake of test consistency